docs/setup/pip.md

```diff
@@ -13,25 +13,25 @@ We support :simple-python: **Python from 3.8 to 3.12**.
     sudo apt-get update -y && sudo apt-get install wget lsb-release -y && \
     CODENAME=$(lsb_release -cs) && \
     TEMP_DEB="$(mktemp)" && \
-    wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/10.2.3/khiops-core-openmpi_10.2.3-1-${CODENAME}.amd64.deb" && \
+    wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/{{ KHIOPS_VERSION }}/khiops-core-openmpi_{{ KHIOPS_VERSION }}-1-${CODENAME}.amd64.deb" && \
     sudo dpkg -i "$TEMP_DEB" || sudo apt-get -f -y install && \
     rm -f $TEMP_DEB && \
-    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/10.2.3.0/khiops-10.2.3.0.tar.gz'
+    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/{{ KHIOPS_PYTHON_VERSION }}/khiops-{{ KHIOPS_PYTHON_VERSION }}.tar.gz'
     ```
 
 
 === "Windows"
     You need to download and install the Khiops desktop application first:
 
-    <a href="https://github.com/KhiopsML/khiops/releases/download/10.2.3/khiops-10.2.3-setup.exe">
+    <a href="https://github.com/KhiopsML/khiops/releases/download/{{ KHIOPS_VERSION }}/khiops-{{ KHIOPS_VERSION }}-setup.exe">
         <button class="btn btn-light btn-sm">
           Download for Windows
         </button>
     </a>
 
     Then, you can run the following Pip command:
     ```sh
-    pip install "https://github.com/KhiopsML/khiops-python/releases/download/10.2.3.0/khiops-10.2.3.0.tar.gz"
+    pip install "https://github.com/KhiopsML/khiops-python/releases/download/{{ KHIOPS_PYTHON_VERSION }}/khiops-{{ KHIOPS_PYTHON_VERSION }}.tar.gz"
```

Comment:
We need to remove "-" from release version in the tarball (see same comment above).